Zotero/Scenes/Detail/ItemDetail/Models/ItemDetailState.swift

```diff
@@ -250,6 +250,7 @@ struct ItemDetailState: ViewModelState {
     var changes: Changes
     var isEditing: Bool
     var isSaving: Bool
+    var isTrash: Bool
```

Comment:
I'd maybe move this to `Data` which already has `isAttachment` just to be consistent.